spec/requests/dashboard_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Dashboard", type: :request do
+  let(:lead_provider) { create(:user, :lead_provider) }
+
+  before do
+    sign_in lead_provider
+    Cohort.create!(start_year: "2021")
```

Comment:
I think this is a bit dangerous here. 2021 seems to be hardcoded quite a lot in the test and we'll spend the first week(s) after Christmas fixing test failures. Could we have `create :cohort, :current` instead or use time travel?